### PR TITLE
GitHub pages.

### PR DESCRIPTION
**Pull request by dvkruchinin**
### Summary

Adding the GitHub pages generation functionality.

### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have added description of my changes into [CHANGELOG](https://github.com/openvinotoolkit/datumaro/blob/develop/CHANGELOG.md)
- [ ] I have updated the [documentation](
  https://github.com/openvinotoolkit/datumaro/tree/develop/docs) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have [linked related issues](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/openvinotoolkit/datumaro/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [x] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2021 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

**Comment by dvkruchinin** [comment]
As an example: https://dvkruchinin.github.io/datumaro/docs/

**Comment by azhavoro** [comment]
> As an example: https://dvkruchinin.github.io/datumaro/docs/

There seems to be an issue with Windows root certificates on our laptops (jQuery cannot be fetched), but it works as expected on any other computer.
image

**Comment by azhavoro** [comment]
@TOsmanov Can we replace jQuery url with Google CDN https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js ?

**Comment by zhiltsov-max** [comment]
I see a broken diagram on the front page

image

**Comment by azhavoro** [comment]
> I see a broken diagram on the front page
> 
> image

It's related with jquery cert issue

**Comment by dvkruchinin** [comment]
I merged changes to my fork. Please take a look https://dvkruchinin.github.io/datumaro/docs/

File: site/build_docs.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+#
+# SPDX-License-Identifier: MIT
+
+import os
+import subprocess
+
+from packaging import version
+import git
+
+MINIMUM_VERSION='1.5.0'
```
Comment (azhavoro) [inline site/build_docs.py:11]: @dvkruchinin Could you change the version to `0.1.11` here? Datumaro has different versions than CVAT.

**Pull request merged**